firestore/src/data_get_as_custom_type.php

```diff
@@ -45,7 +45,7 @@ function data_get_as_custom_type(string $projectId): void
 
     if ($snapshot->exists()) {
         printf('Document data:' . PHP_EOL);
-        print_r($city);
+        print_r((string) $city);
```

Comment:
NIT: Since it's already a string, let's just make this `print` instead of `print_r`
```suggestion
        print((string) $city);
```